package.json

```diff
@@ -16,11 +16,11 @@
     "*.d.ts"
   ],
   "license": "BSD-3-Clause",
-  "source": "src/lru.js",
+  "source": "src/lru.ts",
```

Comment:
we rename `js` to `ts` to be able to generate the type definitions.

Note: Currently its mandatory to port code to TS as `allowJS` and `declaration` options in TS won't work together. This will be address very soon - https://dev.to/open-wc/generating-typescript-definition-files-from-javascript-5bp2, at which point, we can convert this back to `js` if needed.